client/components/TestQueue/index.jsx

```diff
@@ -102,6 +102,7 @@ const TestQueue = () => {
                     <thead>
                         <tr>
                             <th className="test-plan">Test Plan</th>
+                            <th className="test-plan">AT Version</th>
```

Comment:
```suggestion
                            <th className="test-plan">AT Version Requirement</th>
```

That language may be preferred.